Assets/Scripts/Localization/LocalizationTable.cs

```diff
@@ -59,7 +61,7 @@ public static void LoadLocalizationFile(string path)
          * </para>
          * </summary>
          */
-        public static void LoadLocalizationFile(string path, string localizationCode)
+        public static void LoadLocalizationFile(string path, string localizationName)
```

Comment:
@Mizipzor I am not arguing on ```localizationName```. I am agains of changing `Key` in dictionary to be "Display Name" of culture.